### PR TITLE
refactor(testcaseUI): make test case Ui run on multiple platforms

### PR DESCRIPTION
**Pull request by Chreggii**

### CODE REVIEW

**Comment by Chreggii** [comment]
@jsvgoncalves please review :)

File: testcaseUI/testcase-frontend/package.json [hunk 1]
```diff
@@ -89,4 +90,4 @@
     "react-dropdown-select": "^3.6.5",
     "reactstrap": "^8.1.1"
   }
-}
+}
```
Comment (jsvgoncalves) [inline testcaseUI/testcase-frontend/package.json:93]: `\n` missing

File: testcaseUI/testcase-backend/backend/model.py [hunk 2]
```diff
@@ -6,6 +6,7 @@
 from git import Repo
 import os
 import git
+from pathlib import Path
```
Comment (jsvgoncalves) [inline testcaseUI/testcase-backend/backend/model.py:9]: Unused import?

File: testcaseUI/testcase-frontend/package.json (hunk 1 above)
Comment (Chreggii) [reply testcaseUI/testcase-frontend/package.json]: Solved :)

File: testcaseUI/testcase-backend/backend/model.py (hunk 2 above)
Comment (Chreggii) [reply testcaseUI/testcase-backend/backend/model.py:9]: The import was missing before. Therefore I added it :)

**Comment by Chreggii** [comment]
@jsvgoncalves Solved all threads :)

**Pull request merged**